packages/back-end/src/integrations/Postgres.ts

```diff
@@ -34,4 +34,17 @@ export default class Postgres extends SqlIntegration {
   formatDate(col: string) {
     return `to_char(${col}, 'YYYY-MM-DD')`;
   }
+  async runGetSchemaQuery() {
+    const sql = `SELECT
+        table_name,
+        column_name,
+        data_type,
+        table_catalog
+      FROM
+        information_schema.columns
+      WHERE
+        table_schema='public'
+      ORDER BY table_name;`;
+    return await this.runQuery(sql);
```

Comment:
If I set this back to const results = await this.runQuery(sql) can I then define the type as custom instead of the unknown[] that runQuery currently returns?